aldryn_faq/views.py

```diff
@@ -27,11 +28,12 @@ def dispatch(self, request, *args, **kwargs):
 
     def get_context_data(self, **kwargs):
         context = super(FaqMixin, self).get_context_data(**kwargs)
-        context['current_app'] = resolve(self.request.path).namespace
+        context['current_app'] = self.namespace
         return context
 
     def get_queryset(self):
-        return self.model.objects.language(self.current_language)
+        return self.model.objects.language(self.current_language).filter(
+            category__appconfig__namespace=self.namespace)
```

Comment:
how about .filter(category__appconfig=self.config)

The AppConfigMixin provides both self.config and self.namespace